### PR TITLE
Job Positions (vacatures)

### PR DESCRIPTION
**Pull request by StefanKroon**

### CODE REVIEW

**Review by StefanKroon — COMMENTED** [review]
Nothing found. Done

File: Vacatures.html [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]><!--> <html class="no-js"> <!--<![endif]-->
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>TeamUp</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <meta property="og:title" content="TeamUp" />
```
Comment (StefanKroon) [inline Vacatures.html]: Should be TeamTelefoon

File: Vacatures.html [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+    <meta property="og:type" content="image" />
+    <meta property="og:url" content="http://teamup.ask-cs.nl/" />
```
Comment (StefanKroon) [inline Vacatures.html]: Should link to TeamTelefoon. @vijivasanthan Where did you copy this template from? Not from indexTeamTelefoon.html?

File: Vacatures.html [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+            <div class="row-fluid">
+             </div>
+        </div>
+    </div>
+</div>
+</div>
+<!-- end navbar -->
+
+
+<div class="intro" >
```
Comment (StefanKroon) [inline Vacatures.html]: There is too much "empty space" above the "Vacatures" title.

File: Vacatures.html [hunk 4]
```diff
@@ -0,0 +1,301 @@
...
+
+    <div class="intro">
+
+        <div class="container-narrow">
+
+            <div class="row-fluid">
+            </div>
+        </div>
+    </div>
+<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
```
Comment (StefanKroon) [inline Vacatures.html]: Are all these <br> really necessary?

File: Vacatures.html [hunk 5]
```diff
@@ -0,0 +1,301 @@
...
+<footer id="footerTeamTelephone">
+    <div class="contactBtn">
+        <button id="moreInfoButton" class="btn btn-large btn-block btn-primary pulse"
+                data-toggle="modal"
+                data-target="#contactModal">Meer informatie</button>
+    </div>
+    <p>
+
+        Heemraadssingel 89 | 3022 CA Rotterdam | Nederland | 085 888 4445 |
+        <a style="font-size: 16px" href="mailto:contact@teamtelefoon.nl">contact@teamtelefoon.nl</a></p>
```
Comment (StefanKroon) [inline Vacatures.html]: Why is there a seperate font-size tag necessary here? Can't do without?

File: promo/js/main.js [hunk 6]
```diff
@@ -15,6 +15,48 @@ jQuery(function() {
...
+
+
+
+  jQuery('.job2').click(function () {
+
+    jobclassName = jQuery(this).attr('class');
+    vacatureItem = '';
+
+    if (jobclassName == 'job2' && vacatureItem == undefined || vacatureItem == '') {
+      jQuery('#TextContainer1').hide();
```
Comment (StefanKroon) [inline promo/js/main.js]: I think "TextContainer1" is a bad name. Also "TextContainer2" is not very desriptive

File: promo/js/main.js [hunk 7]
```diff
@@ -15,6 +15,48 @@ jQuery(function() {
...
+      jQuery('#TextContainer2').show();
+      return false;
+    }
+  });
+
+
+  jQuery('.vacatures-item').click(function () {
```
Comment (StefanKroon) [inline promo/js/main.js]: This function should probably close the job position on a second click. The code is never executed, I think.

**Review by StefanKroon — COMMENTED** [review]
@vijivasanthan Can you take a look to my 7 comments?

File: Vacatures.html (hunk 2 above)
Comment (vijivasanthan) [reply Vacatures.html]: I copied it from indexTeamUp.html

**Pull request merged**